src/main/java/seedu/module/model/task/UniqueTaskList.java

```diff
@@ -70,6 +70,13 @@ public void setTask(Task target, Task editedTask) {
         internalList.set(index, editedTask);
     }
 
+    /**
+     * Replaces the tasks in the list by deadline.
```

Comment:
I wouldn't describe sorting as replace task in javadoc.

Reply:
Fixed.